packages/frontend/app/clipboard-history/clipboard-details.tsx

```
@@ -18,23 +19,90 @@ export const ClipboardDetails = memo(({item}: ClipboardDetailsParams) => {
+                "URL protocol": metadata.protocol,
+                "Url hostname": metadata.hostname,
+                "Url path": metadata.pathname,
+                "Url hash": metadata.hash,
+                "Url query": metadata.searchParams,
+            };
         case "path":
-            return {};
+            const defaultFirstData = {
+                "Path": value,
+                "Path length": metadata.length,
+            }
```

Biome (1.9.4)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Unsafe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)